openpgp/write_test.go

```diff
@@ -174,13 +174,10 @@ func TestEncryptWithCompression(t *testing.T) {
 		t.Errorf("error closing WriteCloser: %s", err)
 		return
 	}
-	algo, err := checkCompression(buf, kring[:1])
+	err = checkCompression(buf, kring[:1])
 	if err != nil {
 		t.Errorf("compression check failed: %s", err)
 	}
-	if algo != packet.CompressionZLIB {
-		t.Error("message is not compressed correctly")
-	}
```

Comment:
If you want, you could just change the expected compression algorithm (and keep the `checkCompression` function the same), instead of removing this check entirely? But up to you.